### PR TITLE
Tempo: Default search time range to 15 minutes

### LINKED ISSUE #44008: Tempo: Default search time range to 15 mins
For full backend search, default time range to 15 mins to reduce server load.

### PR DESCRIPTION
**Pull request by connorlindsey**
**What this PR does / why we need it**:
Default to 15 minutes to reduce load on search (for when full backend search is enabled)

**Which issue(s) this PR fixes**:

Fixes #44008

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/datasource/tempo/QueryEditor/QueryField.tsx [hunk 1]
```diff
@@ -133,20 +133,6 @@ class TempoQueryFieldComponent extends React.PureComponent<Props, State> {
             />
           </InlineField>
         </InlineFieldRow>
-        {query.queryType === 'nativeSearch' && (
```
Comment (connorlindsey) [inline public/app/plugins/datasource/tempo/QueryEditor/QueryField.tsx:136]: No idea why these were in two separate conditionals before.

**Review by joe-elliott — APPROVED** [review]
thx connor!

**Comment by marefr** [comment]
If this is going to be included in 8.3.4 it needs to be backported, https://github.com/grafana/grafana/blob/main/contribute/merge-pull-request.md#should-the-pull-request-be-backported. Currently no-backport specified.

**Comment by aocenas** [comment]
I am not really sure about this. We have this sentence in the beta disclaimer `It ignores the time range picker` is that not true anymore? If it is that I don't really see why would change the time range. If it isn't true and the time range is respected then this is really an unexpected behavior that we don't have anywhere else. Just changing the time range when you change query type can be really confusing IMHO.

**Comment by connorlindsey** [comment]
For full backend search, that is no longer true. If the backend search feature flag is enabled, that part of the disclaimer is removed.

**Pull request closed without merging**